msr_rr_cfp.md

```diff
@@ -0,0 +1,83 @@
+* Confounding Variable(s) and how their effect will be controlled (e.g., gender might be a confounding factor; we control for it by separating our sample additionally into male and female and using an ANOVA (normal distribution) or Friedman (non-normal distribution) to distill its effect)
+
+For each variable, you should give:
+- name (e.g., presence of family)
+- abbreviation (if you intend to use one)
+- description (whether the family of the crew members travels on board)
+- scale type (nominal: either the family is present or not)
+- operationalization (crew members without family on board vs. crew members with family onboard)
+
+# Material/objects (required)
+Describe any material that you plan to use, be specific on whether you developed it (and how) or whether it is already defined (e.g., a standard myers-briggs-type indicator)
+
+**Example**: For sick days, we recruit the medical records from sick bay (ethics approval pending). For efficiency, we conduct standard interviews with the superior officer and crew members. The questions are the following: / can be found on the Web site / Appendix Furthermore, we observe their performance during a simulated task.
+
+# Tasks (optional)
+If you use tasks, describe them, how they were designed or from where they are taken and why they are suitable to evaluate the hypotheses / research question
+
+**Example**: For effectiveness of the crew members, we ask them to sweep a class 2 nebular. We simulate an error in the primary sensory array. Crew members should then run a level 3 diagnostic to spot the error, fix the error, and complete the sweep of the nebular. We measure the time to (i) spot that there is an error, (ii) decide on the correct diagnostic protocol, (iii) fix the error, and (iv) complete the sweep.
+
+# Participants/Subjects/sample (required)
+Describe how and why you select the sample. When you conduct a meta analysis, describe the primary studies / work on which you base your meta analysis
+
+**Example**: We recruit crew members from the science department on a voluntary basis. They are our targeted population. 
+
+# Execution Plan (required)
+Describe the experimental protocol
+
+**Example**: Each crew member needs to sign the informed consent and agreement to process his or her data according to GDPR. Then, we conduct the interviews. Afterwards, participants need to complete the simulated task. 
+
+# Analysis Plan (required)
+## Descriptive statistics
+How do you describe the data? How do you handle outliers?
+
+Example: To represent the number of sickdays, we use histograms. Dependending on the distribution, we remove values that are 2 standard deviations above the mean as outliers (normal distribution). If the data are non-normal, we use the median and values below the 10th/above the 90th percentile.
+
+## How do you evaluate the hypotheses?
+Are you using significance tests? How are you avoiding inflation of type-I error (if applicable)
```

Comment:
I'd like this to be written in a way that also more advanced (IMHO) statistical procedures (Bayesian, for example) are also allowed/ok and so that we don't imply a bias for classical, Fisherian, parametric analysis methods/approaches.

Reply:
:) Maybe better to say less here. In general the OSF guide is pretty vague, while we are very detailed - people might get confused.

Reply:
How about being more general so that other statistical procedures are implied? Something like "How are you testing the significance of your results?"